x-pack/plugins/integration_assistant/server/routes/analyze_logs_routes.ts

```diff
@@ -106,11 +106,21 @@ export function registerAnalyzeLogsRoutes(
 
           switch (graphLogFormat) {
             case 'unsupported':
-              throw new UnsupportedLogFormatError(
-                GenerationErrorCode.UNSUPPORTED_LOG_SAMPLES_FORMAT
-              );
+              throw new UnsupportedLogFormatError({
```

Comment:
could we add some tests for these scenarios & this file in general?

It looks like we currently have none

Reply:
Yeah.. I think this file needs tests on the whole.. probably will push a followup PR with tests added